src/validators/courseContentValidator.ts

```diff
@@ -0,0 +1,48 @@
+import {IsNotEmpty, MaxLength, validate, ValidationError} from 'class-validator'
```

Comment:
INFO - Are we using this `https://www.npmjs.com/package/class-validator`? If so, why are we creating one property classes for just validation and not annotate the Course class?

Reply:
Because the UI split the creation of the course over multiple screens. On the first screen the user enters the course title. On the second screen the user add description, short description and learning outcomes. Obviously it would be a lot cleaner to the validation in one place before we save the course. Unfortunately the UI makes it difficult to do this.